lib/ember_cli/build_monitor.rb

```diff
@@ -32,7 +32,7 @@ def wait!
     attr_reader :name, :paths
 
     def complete?
-      !paths.lockfile.exist?
+      !paths.lockfile.exist? && paths.dist.join("index.html").exist?
```

Comment:
The addon no longer copies the `index.html` file:

https://github.com/rondale-sc/ember-cli-rails-addon/commit/fd6d6bac28758d6700012bd12d02d1d25a4c9156

Copying the index was a means of integrating with Sprockets, and is no longer necessary.